sos/collector/transports/__init__.py

```diff
@@ -189,7 +196,8 @@ def _format_cmd_for_exec(self, cmd):
                         transport
         :rtype:         ``str``
         """
-        cmd = "%s %s" % (self.remote_exec, quote(cmd))
+        cmd = "%s %s" % (self.remote_exec,
+                         quote(cmd) if self.quote_cmds else cmd)
```

Comment:
`self.quote_cmds` is a dead variable, always set to `True`?

Reply:
This is a bit of a left-over from an earlier draft - initially I was using `oc debug` which required some "interesting" quoting work. At the moment yes it's a dead variable but I wonder if it would be useful for future transports (of course, in that case we can always add it back at that time).

I'll pull it out in the next branch update.